src/compass/s1_geocode_metadata.py

```diff
@@ -115,9 +119,9 @@ def run(cfg, burst, fetch_from_scratch=False):
                 dtype = np.byte
 
             # Create dataset with x/y coords/spacing and projection
-            topo_ds = init_geocoded_dataset(static_layer_group, layer_name,
-                                            geo_grid, dtype,
-                                            np.string_(layer_name))
+            topo_ds = initdata_geocoded_dataset(static_layer_group, layer_name,
```

Comment:
```suggestion
            topo_ds = init_geocoded_dataset(static_layer_data_group, layer_name,
```
Also, for consistency, we should use `static_layers_data_group`